advocacy_docs/partner_docs/ThalesCipherTrustManager/05-UsingThalesCipherTrustManager.mdx

```diff
@@ -0,0 +1,191 @@
+---
+title: 'Using'
+description: 'Walkthrough of example usage scenarios'
+---
+
+After you have configured all of the Thales CipherTrust Manager keys as stated in the Configuring section, you will be able to use them in conjunction with EDB Postgres Advanced Server 15.2.
+
+!!! Note
+    It is important to note that this doc is intended for versions 15.2 of EDB Postgres Advanced Server as this version has the necessary addition of TDE.
```

Comment:
See note in Hashicorp.   Should be the same.
We should also to a compare as notes like this and other areas that are exact info should match.